crates/providers/Cargo.toml

```diff
@@ -45,6 +46,8 @@ tracing = { version = "0.1", default-features = false }
 tracing-futures = { version = "0.2", default-features = false }
 tracing-opentelemetry = { version = "0.20", default-features = false }
 tracing-subscriber = { version = "0.3", default-features = false }
+url = { version = "2.4.1", default-features = false }
+vaultrs = { version = "0.7.0", default-features = false }
```

Comment:
Ahh this happens to be on purpose --  `crates/providers` is [ *excluded* from the top level workspace](https://github.com/wasmCloud/wasmCloud/blob/main/Cargo.toml#L70), it's it's own thing since to avoid nesting since it's [not supported yet](https://github.com/rust-lang/cargo/issues/5042). 

Changing that causes this warning:

```
➜ cargo build
warning: /path/to/wasmCloud/crates/providers/kv-vault/Cargo.toml: dependency (vaultrs) specified without providing a local path, Git repository, version, or workspace dependency to use. This will be considered an error in future versions
```